app/views/dashboard/payments/edit.html.haml

```diff
@@ -0,0 +1,80 @@
+- content_for(:page_title) { 'Payment Information' }
```

Comment:
This is HAML and it represents an HTML page. HAML, once you're used to its syntax, is simpler and less error-prone than plain HTML because it makes it impossible to mix up your closing tags (`</a>`) or forget to include them.